text/0003-multisig-minting/0003-multisig-minting.md

```diff
@@ -32,14 +32,16 @@ Adding a simple rule for multi-signature scripts that blocks should be signed by
 
 ## Advantages
 
-* More secure minting.
-* Permanent outsourcing of minting is not possible, every mint needs to be pre-signed by the holder.
-* Minimal protocol changes.
+* Enabling multiple-signature addresses to mint. While the coinstake would need to be signed by the required number of signatures, the block would only need to be signed by any one member of the multi-sig regardless of the number of signatures required for a normal transaction. 
+* Allowing users to only access their wallets once every maturation period (30 days in the current protocol) while still minting continuously, instead of having to give continuous wallet access to their clients.
+* Allowing mint pools to exist, though this will require third party infrastructure to navigate the periodic minting process.
 
 ## Drawbacks
 
-* Hard fork
+* Dependency on other RFCs with drawbacks.
+* The convention of using just one of the multi-sig keys to sign the block must be changed if alternatives are prefered in the future.
+* Mint pools have existential governance issues, similar to mining pools, in which much of the blockchain governance is transfered to the pool rather than the coin holder.  While this proposal is expected to increase minter participation, it is questionable about whether that participation is truly 'distributed'.
```

Comment:
I don't believe this changes compared to today. Today minters vote by choosing a client implementation, with this change some minters would vote by choosing a pool, I don't see any differences here.

Reply:
The pool operator has the power to e.g. blackball certain transactions or choose to orphan certain blocks without the participants even being aware of it.  Voting for a representative is not necessarily the same as direct democracy.

Reply:
Today voters also trust the client developers they implement what they say which is similar to voting for a representative (being the client implementation).
Pools are transparent on what they do on chain because of it's public nature, the threshold to validate the pool's behavior is equal to validating the client implementation behavior.
The only difference I can see is that a pool can update it's behavior without notice.

Reply:
What they put on-chain is public, but what they decide not to put can only be inferred.  While a particlarly diligent peercoiner can compile their own source code and know what they are running on their own machine, such diligence is not possible with a pool.  So there are two governance issues, the first being that as you say the pool can always misbehave with no notice to the owner of the coins.  The second is that they could prevent certain transactions from being included or orphan certain blocks.  If done infrequently, it could be impossible to determine coincidence versus malice, as these decisions occur off-chain.

Either way, governance via a pool as opposed to on individual hardware has both benefits and detriments.  We should state them both.

Reply:
Agreed, this modification is fine for me.